packages/containers/src/HeaderBar/HeaderBar.container.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
+import { Map } from 'immutable';
+import { cmfConnect, Inject } from '@talend/react-cmf';
+
+export const DEFAULT_STATE = new Map({});
+
+class HeaderBar extends React.Component {
+	static displayName = 'Container(HeaderBar)';
+
+	static PRODUCTS_COLLECTION_ID = 'header_bar_products_collection';
+
+	static propTypes = {
+		products: PropTypes.shape({
+			url: PropTypes.string,
+			lang: PropTypes.string,
+			env: PropTypes.string,
```

Comment:
After further reflection, this may not be necessary since `env` could be already reflected through requested endpoint (domain ...). Will update as soon as I know.